src/web/mjs/connectors/MangaReaderTo.mjs

```diff
@@ -83,12 +83,16 @@ export default class MangaReaderTo extends Connector {
 
         const dom = this.createDOM(data.html);
         const imagesArr = Array.from(dom.querySelectorAll('.iv-card'));
+        const shuffled = imagesArr.length > 0 ? imagesArr[0].className.includes('shuffled') : false;
+
+        if (!shuffled)
```

Comment:
May be simplified to remove intermediate variable `shuffled`

```javascript
if(!imagesArr.length || !imagesArr[0].className.includes('shuffled')) {
    // ...
}
```

Reply:
Fixed